book/models.py

```diff
@@ -1,3 +1,7 @@
 from django.db import models
-
-# Create your models here.
+from user.models import UserEmployee
+class Book(models.Model):
+    name = models.CharField(max_length=50,null=False)
+    price = models.DecimalField(max_length=10,decimal_places=2)
+    creator = models.ManyToManyField(UserEmployee)
```

Comment:
f.k.

Reply:
sir i added but for now i did't upload model latest file.